### PR TITLE
[Task] Init resource management

### PR DESCRIPTION
**Pull request by truyentd-2427**
## What's this PR do ?

- Config ```FlutterGen```, ```flutter_localizations```

## Checklist

- [ ] Write/Update Unit Test (if have)
- [ ] Fill link PR into ticket and the opposite
- [ ] Validate UI/Model/API with latest specs
- [ ] Add UI pics to this PR (if have).

## Library

- FlutterGen: https://pub.dev/packages/flutter_gen
- Localizations: https://docs.flutter.dev/ui/accessibility-and-localization/internationalization

## Demo

https://github.com/namnh-0652/flutter_movie_clean/assets/77828973/10d7d512-db87-4b73-b4b0-56dc6ec9b497

### CODE REVIEW

File: lib/gen/assets.gen.dart
Comment (namnh-0652) [inline lib/gen/assets.gen.dart:1]: Add `*.gen.dart` to gitignore file

File: lib/l10n/l10n.dart [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+extension AppLocalizationsExtensions on BuildContext {
```
Comment (namnh-0652) [inline lib/l10n/l10n.dart]: should be `BuildContextExt`, we will use more extension functions from this context

File: assets/images/ic_plus.svg
Comment (namnh-0652) [inline assets/images/ic_plus.svg:1]: Is this unused?

**Review by ghost — APPROVED** [review]

File: assets/images/ic_plus.svg
Comment (truyentd-2427) [reply assets/images/ic_plus.svg:1]: This icon is used to show how to use the generated assets. Please look at [this line](https://github.com/namnh-0652/flutter_movie_clean/pull/33/files#diff-e61eb31d013d12616f5532636a88cfa63631dda8f7829e5424e68542214d1608R87)

**Review by namnh-0652 — APPROVED** [review]

**Pull request merged**